web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -98,6 +98,13 @@
  *   We add a new event handler, resizeHandler, for window.on('resize', ...)
  *   that calls this.show to re-render the typeahead in the correct position.
  *
+ * 10. Allow typeahead to be located next to its searchbar in the DOM
+ *
```

Comment:
This line should probably be more generic (something like "Manually position typeahead only when it's `parentElement` is not specified") and not focus on the 1 use case. The fact that it's used for the search dropdown can be mentioned in the paragraph below.

Reply:
Oh yeah of course, oops.

Reply:
Ohhh I remember, what I meant to express was beside the input field. I can change to that.